### ISSUE TITLE
Move ndarray and related packages to a separate org?

### ISSUE THREAD

**Issue by dy** [scijs/contributing#29]
Maybe I am saying blasphemous things, but wouldn't it be better for _ndarray_ and _ndarray-*_ related packages to reside under _github.com/ndarrayjs_ org?

I mean it would be way better UX to have ndarray-related modules under the same roof and keep scijs for generic-type modules, for both who use _ndarrays_ and who use generic scientific packages.

Especially the utility packages, like [ndarray-pack](https://github.com/scijs/ndarray-pack), [ndarray-fill](https://github.com/scijs/ndarray-fill), [cwise-compiler](https://github.com/scijs/cwise-compiler), [cwise-parser](https://github.com/scijs/cwise-parser) etc. - they have very little to do with science.

Focusing org on a single project seems to be a good practice in [regl-project](https://github.com/regl-project/), [gulp](https://github.com/gulpjs), [d3](https://github.com/d3) etc.

@rreusser @mikolalysenko @scijs/owners

**Comment by rreusser** [comment]
I tend to agree. Would love to clean house. Maybe time to tackle https://github.com/scijs/ndarray/issues/18 then? Would also love to clean up and drop some of the lesser modules while we're at it.

**Comment by dy** [comment]
For :wastebasket: modules there is [scrapjs](https://github.com/scrapjs)

**Comment by rreusser** [comment]
Ooh. Good catch. [ndarray-dtype](https://github.com/mattdesl/ndarray-dtype) would be good to really think through and work in, if applicable. I guess specifically I'm thinking of python's `numpy.array([...], dtype='uint8')` and [`astype`](https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.ndarray.astype.html) that really make it quick and easy to work with types. Should think through common js use cases and whether the api can be streamlined at all.

**Comment by mikolalysenko** [comment]
I agree with this general sentiment, but don't have attention right now to take on a project like this.
